components/landing/SignupBanner.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="columns is-multiline">
+    <div class="column is-8-desktop is-6-tablet is-12-mobile">
+      <div class="columns is-multiline">
+        <div
+          class="column is-6-desktop is-12-tablet is-12-mobile is-flex is-align-items-center">
+          <h3 class="is-size-4 has-text-weight-bold">
+            {{ $t('signupBanner.title') }}
+          </h3>
+          <img src="/signup-voucher.svg" alt="signup voucher" />
+        </div>
+
+        <div
+          class="column is-6-desktop is-12-tablet is-12-mobile is-flex is-align-items-center">
+          <span>
+            {{ $t('signupBanner.subtitle') }}
+          </span>
+        </div>
+      </div>
+    </div>
+
+    <div
+      class="column is-4-desktop is-6-tablet is-12-mobile is-flex is-align-items-center">
+      <form class="is-flex is-align-items-center" @submit.prevent="submit">
+        <NeoInput
+          v-model="email"
+          type="email"
+          class="fixed-height"
+          placeholder="jane.doe@kodadot.xyz"
```

Comment:
I dont have any issues , but how would this conflict with the existing signup form ?
 for testing purposes  ?

Reply:
oh, its just because there is a getbyplaceholder in the playwright test that checks the footer, it finds the element by the jane.doe@kodadot.xyz placeholder, and at the time I've checked this PR they were conflicting when tests were running, but now it seems to be running without any conflicy, strange, I'll remove this comment.